### DISCUSSION TITLE
Can't enable github copilot after disabling globally in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by gmanvel** [community/community#41076] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I disabled github copilot & can't enable it back for Visual Studio 2022 Version 17.4.2. - https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-visual-studio#enabling-or-disabling-github-copilot

Enable options are disabled. Am I missing something here ?

![copilot](https://user-images.githubusercontent.com/11509911/206427333-03fe951f-62c9-4402-84b5-bcd8c3ddc387.jpg)

**Comment by Akash1134** [comment]
Hi @gmanvel, Sorry that you are facing issues enabling Copilot for VS 2022. Could you please try signing out? Copilot will prompt you to sign back in.

**Reply by gmanvel** [reply]
Thank you for suggestion @Akash1134. I'm logged in to VS via different account. How do I sign out from Copilot ? I don't see such option. Also I'm unable to delete copilot extension, to try to reinstall and see if issue will resolve itself.

![image](https://user-images.githubusercontent.com/11509911/206513287-58a6e00c-97e5-44b8-97d4-45630c2d46f1.png)

**Reply by BlLogan** [reply]
Hi, did you find a solution for this problem?

**Reply by gmanvel** [reply]
Unfortunately no. I ended up uninstalling Visual Studio & installing everything again.

**Comment by ilFusta** [comment, marked as answer]
I managed to find a solution, you can enable or disable Copilot globally from the "Tools" -> "Options" menu and search for Copilot in the search box.

![Screenshot (3)](https://user-images.githubusercontent.com/13522564/207321746-dc797f67-aa20-4bd1-863f-24f650e16178.png)

**Reply by gmanvel** [reply]
Thank you for this. So once you enable again, is it manageable from left bottom corner menu item ?
![image](https://user-images.githubusercontent.com/11509911/207396011-4a87d557-085e-4c12-97d8-36aaec10baab.png)

**Reply by ilFusta** [reply]
yes exactly.

**Reply by AnMaMo** [reply]
i have tried test to uninstall and reinstall Github copilot extension, and this doesn't work, but your answer solved my problem, thanks!

**Reply by Sarthakpurii** [reply]
where i can find tools option?
